docs/learn/resource-materials/fact-base.md

```diff
@@ -1,34 +1,15 @@
 ---
 title: Fact Base
-description: A collection of fact's to conquer common misunderstandings and false proclamations bout IOTA, Get your fact's right!
+description: A collection of fact's to conquer common misunderstandings and
+  false proclamations bout IOTA, Get your fact's right!
 ---
 
+# Fact Base
 
-# Heading 1 (Page Title)
+We are looking for contributors that would like to put together a fact base for IOTA.
 
-content - introduction to the topic
+If You feel you can participate in this topic, please use the editor button below and start contributing.
 
+### Thanks a lot[#](https://wiki.iota.org/docs/learn/networks/network-token-migration#thanks-a-lot)
```

Comment:
What is the idea of this link? Can we remove it?
![Screenshot_3](https://user-images.githubusercontent.com/53269239/133845257-619a786c-9cbb-4686-9393-df7265a8b99d.jpg)

Reply:
ah yes, they are useless, ill remove it